### PR TITLE
Add longoptions support for psconvert.

### PR DESCRIPTION
**Pull request by rbdavis**
**Description of proposed changes**

Python differences:

  No differences.

Julia differences:

  Replaced -I icc_gray (???) with python resize.
  Julia does not have -N
  Some other aliases added on top of Julia.

Fixes #

**Reminders**

- [ ] Make sure that your code follows our style. Use the other functions/files as a basis.
- [ ] Add tests for new features or tests that would have caught the bug that you're fixing.
- [ ] Describe changes to function behavior and arguments in a comment below the function declaration.
- [ ] If adding new functionality, add a detailed description to the documentation and/or an example.

### CODE REVIEW

**Review by PaulWessel — APPROVED** [review]

**Comment by PaulWessel** [comment]
@remkos, make sure you dont remove aliases that the python and Julia folks have put in since they are needed.

**Comment by rbdavis** [comment]
Also restored the Python -N bb_style string removed by yesterday's changes.

File: src/longopt/psconvert_inc.h [hunk 1]
```diff
@@ -25,74 +25,45 @@ static struct GMT_KEYWORD_DICTIONARY module_kw[] = {
...
-	          "",                  "",
-	          "",                  "",
-		  GMT_TP_STANDARD },
-	{ 0, 'W', "",
-	          "",                  "",
-	          "",                  "",
-		  GMT_TP_STANDARD },
-	{ 0, 'Z', "",
-	          "",                  "",
+	{ 0, 'W', "esri|world_file",
```
Comment (joa-quim) [inline src/longopt/psconvert_inc.h:61]: I don't see the need to add `esri`, which is the name of the company. `world_file` is like GDAL refers to it for example.

File: src/longopt/psconvert_inc.h [hunk 2]
```diff
@@ -25,74 +25,45 @@ static struct GMT_KEYWORD_DICTIONARY module_kw[] = {
...
-	{ 0, 'M', "",
-	          "",                  "",
+	{ 0, 'L', "listfile|list_file", "", "", "", "", GMT_TP_STANDARD },
+	{ 0, 'M', "pslayer|ps_layer",
+	          "b,f",               "bg|background,fg|foreground",
 	          "",                  "",
 		  GMT_TP_STANDARD },
-	{ 0, 'N', "",
-	          "",                  "",
+	{ 0, 'N', "bb_style|show_bbox",
```
Comment (joa-quim) [inline src/longopt/psconvert_inc.h:47]: I prefer `bgcolor`. This is mainly not about the BoundingBox per se but specially on the possibility to paint the BB area.

**Comment by joa-quim** [comment]
Applied suggested changes.

**Pull request merged**